docs/rdd-programming-guide.md

```diff
@@ -276,7 +276,7 @@ $ PYSPARK_DRIVER_PYTHON=jupyter PYSPARK_DRIVER_PYTHON_OPTS=notebook ./bin/pyspar
 
 You can customize the `ipython` or `jupyter` commands by setting `PYSPARK_DRIVER_PYTHON_OPTS`.
 
-After the Jupyter Notebook server is launched, you can create a new "Python 2" notebook from
+After the Jupyter Notebook server is launched, you can create a new notebook from
 the "Files" tab. Inside the notebook, you can input the command `%pylab inline` as part of
 your notebook before you start to try Spark from the Jupyter notebook.
```

Comment:
BTW, Jupyter with PySpark in this way seems broken in Spark due to how [find-spark-home](https://github.com/apache/spark/blob/master/bin/find-spark-home) works in PySpark, see https://github.com/apache/spark/pull/28256

Reply:
It's been long time not using pyspark, I'm glad that this is fixed in spark 3.0. Thanks.